angular-register.ts

```diff
@@ -0,0 +1,30 @@
+declare interface NoopDecorator {
+  <T>(type: T): T;
+  (): (...args: any[]) => void;
+}
+
+declare global {
+  /**
+   * Indicates that this function or variable is being overridden
+   * from the implemented or extended parent class.
+   *
+   * @see [TSLint Override](https://github.com/hmil/tslint-override)
+   */
+  // tslint:disable-next-line: variable-name
+  const Override: NoopDecorator;
```

Comment:
This needs to be a `var` and not a `const` or `let` to avoid this error  on line  25
![Screen Shot 2020-01-16 at 19 07 25](https://user-images.githubusercontent.com/8206108/72575863-7efbca80-3893-11ea-9f5d-e21ebcc98bed.png)